src/vue/Alert.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    v-if="show"
+    transition="fade"
+    class="uk-alert"
+    :class="{
+      'uk-alert-success': style === 'success',
+      'uk-alert-warning': style === 'warning',
+      'uk-alert-danger': style === 'danger',
+      'uk-alert-large': size === 'large'
+    }">
+    <a href=""
+      class="uk-close uk-alert-close"
+      v-if="close"
+      @click.prevent="toggleMessage()">
+    </a>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import UI from 'uikit'
+import Vue from 'vue'
+
+export default {
+  props: {
+    style: {
+      type: String,
+      default: ''
+    },
+    size: {
+      type: String,
+      default: ''
+    },
+    close: {
+      type: Boolean,
+      default: true
+    },
+    show: {
+      type: Boolean,
+      default: true
+    },
+    transition: {
+      type: Object,
+      default: function () {
+        return {
+          duration: 200,
+          fade: true
+        }
+      }
+    }
+  },
+  computed: {
+    getTransitionOptions: function () {
```

Comment:
its not a method, but a computed property, so dont add an action in the name. Rename it to `transitionOptions`.